src/Microsoft.OData.Edm/Validation/EdmErrorCode.cs

```diff
@@ -1388,67 +1388,57 @@ public enum EdmErrorCode
         /// </summary>
         AnnotationApplyToNotAllowedAnnotatable = 400,
 
-        #region JSON
+        /// <summary>
+        /// Invalid $Key value.
+        /// </summary>
+        InvalidKeyValue,
 
         /// <summary>
-        /// Invalid JSON
+        /// Invalid mulitple entity container in one schema
         /// </summary>
-        JsonInvalid,
+        SchemaCannotHaveMoreThanOneEntityContainer,
 
         /// <summary>
-        /// The specified version number is not valid.
+        /// Invalid $Collection value.
         /// </summary>
-        JsonInvalidVersionNumber,
+        InvalidCollectionValue,
+
+        #region JSON
 
         /// <summary>
-        /// Invalid JSON value
+        /// Invalid JSON
         /// </summary>
-        JsonInvalidValue,
+        InvalidJson,

```

Comment:
>InvalidJson [](start = 8, length = 11)

I'm okay with this one, as it is specific to JSON. Alternative would be to say "InvalidPayload" or "InvalidPayloadForFormat".